src/map.cpp

```diff
@@ -102,7 +102,7 @@ MapSector * Map::getSectorNoGenerateNoLock(v2s16 p)
 		return sector;
 	}
 
-	std::map<v2s16, MapSector*>::iterator n = m_sectors.find(p);
+	std::unordered_map<v2s16, MapSector*>::iterator n = m_sectors.find(p);
```

Comment:
```suggestion
	auto n = m_sectors.find(p);
```
much more portable :)